stories/input.js

```diff
@@ -109,13 +109,13 @@ const ExampleInput = () => {
         id="id8"
         value={numberValue}
         onChange={(e) => {
-          if (e && e.target) setNumberValue(e.target.value);
+          if (e && e.target) setNumberValue(e.currentTarget.value);
```

Comment:
I don't think it's needed to check the existence of the event anymore since we added the typing. WDYT?

```suggestion
        setNumberValue(e.currentTarget.value);
```